kernel/classes/datatypes/ezuser/ezuser.php

```diff
@@ -27,6 +27,8 @@ class eZUser extends eZPersistentObject
     const PASSWORD_HASH_MYSQL = 4;
     /// Passwords in plaintext, should not be used for real sites
     const PASSWORD_HASH_PLAINTEXT = 5;
+    /// Passwords in bcrypt of password
+    const PASSWORD_BCRYPT = 6;
```

Comment:
@michalpipa Here @pedroresende is explicitly using php's PASSWORD_BCRYPT constant and not PASSWORD_DEFAULT. I was thinking of suggesting to change that, but I think I prefer this as otherwise users using different PHP versions in production and dev might get into situation where their database contains future hashes not supported by given PHP version.

So it's ok that he specifies as long as we are specific to `password_hash` as well.